core/renderers/block_rendering_rewrite/block_render_info.js

```diff
@@ -429,6 +458,10 @@ Blockly.blockRendering.RenderInfo.prototype.getInRowSpacing_ = function(prev, ne
     return Blockly.blockRendering.constants.LARGE_PADDING;
   }
 
+  if (next.isJaggedEdge()) {
```

Comment:
Add a comment: between anything and a jagged edge.

Reply:
Done